markdown/developers/sync.md

```diff
@@ -46,7 +46,7 @@ Sorry about this, but there's no way around it..
 You should not be actively working on the main nf-core repository, so we need to bring these changes to your personal fork.
 The steps we need to do are:
 
-1. Pull the `nf-core/<pipeline>` `TEMPLATE` changes to your fork
+1. Pull the `nf-core/<pipeline>` `TEMPLATE` changes to your fork (or the `nf-core-template-merge-<version>` branch)
```

Comment:
Should be identical, so no need to complicate things by mentioning the branch here I think.

Could be good to give example commands here though:

```bash
git remote add upstream <url>
git fetch upstream
git checkout -b merge-template-updates
git merge upstream/TEMPLATE
# [..fix merge conflicts, git add changes..]
git commit -m "Fixed merge conflicts"
git push --set-upstream origin merge-template-updates
# Open pull-request from your fork
```

Reply:
ah fail, this is already in the document just below 🤦🏻